include/bluetooth/mesh/models.h

```diff
@@ -0,0 +1,74 @@
+/** @file
+ *  @brief Bluetooth Mesh Models API.
+ */
+
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_MESH_MODELS_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_MESH_MODELS_H_
+
+/* GENERIC BATTERY LEVEL (0x100C) */
+
+/* Generic Battery Level defines */
+#define MODEL_BATTERY_SRV_CHARGE_LEVEL_UNKNOWN 0xFF
+
+/* Generic Battery Time To Discharge defines */
+#define MODEL_BATTERY_SRV_UNKNOWN_DISCHARGE_TIME 0xFFFFFF
+
+/* Generic Battery Time To Charge defines */
+#define MODEL_BATTERY_SRV_UNKNOWN_CHARGE_TIME 0xFFFFFF
+
+/* Generic Battery Flags */
+/* Generic Battery Flags Presence */
+#define MODEL_BATTERY_SRV_FLAG_PRES_NOT_PRES     (0 << 0)
```

Comment:
BIT(0)

Reply:
Ok

Reply:
Actually, what's going on here? These don't look like flags since your'e shifting other things than `1`

Reply:
Actually I was influenced by MESH Model 1.0 Specification, they call those 2 bit values "flags".

Reply:
> Ok

@kolodgrz no, not ok :)

The code comment said flags, so I assumed that's what these are. Instead it looks like this is an enumeration that occupies the lower 3 bits of a 8-byte integer?

Reply:
"Bit Definition
0-1 Generic Battery Flags Presence
2-3 Generic Battery Flags Indicator
4-5 Generic Battery Flags Charging
6–7 Generic Battery Flags Serviceability"

Generic battery Flag Presence (for example):
"Value Description
0b00 The battery is not present.
0b01 The battery is present and is removable.
0b10 The battery is present and is non-removable
0b11 The battery presence is unknown."